scripts/ci-multidisk.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
This isn't actually a test, it was just setting up the bundle store before we ran test-cli. This predates the removal of SingleDisk before Multi was the default, so if we go ahead and delete this script we should be fine.